.travis.yml

```diff
@@ -28,148 +28,157 @@ env:
-        packages:
-        - libkrb5-dev
-        - libnorm-dev
-        - libpgm-dev
-        - libsodium-dev
-        - libnss3-dev
-        - libbsd-dev
-  - env: BUILD_TYPE=default CURVE=libsodium DRAFT=enabled
-    os: osx
-  - env: BUILD_TYPE=default CURVE=libsodium DRAFT=enabled ADDRESS_SANITIZER=enabled
-    os: linux
-    addons:
-      apt:
-        packages:
-        - libsodium-dev
-  - env: BUILD_TYPE=android CURVE=tweetnacl
-    os: linux
-    dist: trusty
-  - env: BUILD_TYPE=cmake DO_CLANG_FORMAT_CHECK=1 CLANG_FORMAT=/usr/lib/llvm-8/bin/clang-format
-    os: linux
-    addons:
-      apt:
-        packages:
-        - clang-format-8
-  - env: BUILD_TYPE=default POLLER=poll
-    os: linux
-  - env: BUILD_TYPE=default POLLER=select
-    os: linux
-  - env: CXX=clang++ BUILD_TYPE=default CURVE=libsodium GSSAPI=enabled PGM=enabled NORM=enabled FORCE_98=enabled
-    os: linux
-    compiler: clang
-    addons:
-      apt:
-        sources:
-        - sourceline: 'deb http://download.opensuse.org/repositories/network:/messaging:/zeromq:/git-stable/xUbuntu_14.04/ ./'
-          key_url: 'http://download.opensuse.org/repositories/network:/messaging:/zeromq:/git-stable/xUbuntu_14.04/Release.key'
-        packages:
-        - libkrb5-dev
-        - libnorm-dev
-        - libpgm-dev
-        - libsodium-dev
-  - env: BUILD_TYPE=abi-compliance-checker
-    os: linux
-    dist: xenial
-    addons:
-      apt:
-        packages:
-        - abi-dumper
-        - abi-compliance-checker
-    
+    - if: type = cron OR (branch =~ analyze$ AND type = push)
+      env: BUILD_TYPE=cmake CLANG_TIDY=/usr/bin/clang-tidy-10 CC=clang-10 CXX=clang++-10
+      os: linux
+      compiler: clang
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+            - llvm-toolchain-10
+          packages:
+            - clang-10
+            - clang-tools-10
+            - clang-tidy-10
+    - env: BUILD_TYPE=default CURVE=tweetnacl IPv6=ON
+      os: linux
+      dist: precise
+    - env: BUILD_TYPE=coverage CURVE=tweetnacl GSSAPI=enabled PGM=enabled NORM=enabled TIPC=enabled TLS=enabled DRAFT=enabled
+      os: linux
+      addons:
+        apt:
+          sources:
+            - sourceline: "deb http://download.opensuse.org/repositories/network:/messaging:/zeromq:/git-stable/xUbuntu_18.04/ ./"
+              key_url: "http://download.opensuse.org/repositories/network:/messaging:/zeromq:/git-stable/xUbuntu_18.04/Release.key"
+          packages:
+            - libkrb5-dev
+            - libnorm-dev
+            - libpgm-dev
+            - libgnutls28-dev
+            - lcov
+    - env: BUILD_TYPE=coverage CURVE=tweetnacl GSSAPI=enabled NORM=enabled TIPC=enabled IPv6=ON TLS=enabled DRAFT=enabled
+      os: linux
+      arch: arm64
+      addons:
+        apt:
+          packages:
+            - libkrb5-dev
+            - libnorm-dev
+            - libgnutls28-dev
+            - lcov
```

Comment:
is lcov included intentionally? it makes the job a lot longer

Reply:
I think it's required, but I can try to remove it again and see if it still works.

Reply:
I wonder: How did you see that lcov makes the job a lot longer?

Reply:
it's doing all the coverage things  - gcov files etc

Reply:
Well, yes, it's a coverage job.

Reply:
ahah, sorry, my bad - I though you meant it in the sense of ensuring ipv6 tests run, rather than actual code coverage